apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -1062,11 +1062,11 @@ impl QueryPlan {
 
     fn extract_authorization_metadata(
         &mut self,
-        subgraph_schemas: &SubgraphSchemas,
+        schema: &apollo_compiler::Schema,
```

Comment:
isn't `supergraph_schema()` returnin a `Valid<Schema>`?

Reply:
It does. This first pass just restores the previous type signature, but I can update to use `Valid<Schema>` so we don't need `Valid::assume_valid_ref` anymore

Reply:
Updated in https://github.com/apollographql/router/pull/5047/commits/cf25e1660cafe6eb323845f831a7eb546907c9e3